Sources/StreamChat/Models/ChatMessage.swift

```diff
@@ -18,7 +18,7 @@ public struct ChatMessage {
     public let cid: ChannelId?
     
     /// The text of the message.
-    public let text: String
+    public var text: String
```

Comment:
@tunguyen-zien  is this necessory?

Reply:
yes, going forward, if any card type that we have not supported yet, we will show the fallbackMessage from BE instead of the raw msg (ex: Pls upgrade your app to the new version)